.github/workflows/on_push.yml

```diff
@@ -0,0 +1,34 @@
+name: Run Pylint and tests
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10"]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v3
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
```

Comment:
Is it possible to use something like `pip install -e .` to also check `setup.py`?

Reply:
@mikulskibartosz Could you response to the question?

Reply:
I think so. How would you like to test it? What do you want to check after installing the package?